vocab/security/template.html

```diff
@@ -260,16 +256,19 @@ <h2>Diagram description</h2>
           The diagram uses boxes, ellipses, and connecting lines with different "styles" 
           (border color, end marker, line type) to differentiate their semantic meaning; 
           these styles identify Property, Class, or Datatype, via the shapes used for the 
-          graph nodes, and Superclass, Domain Of, Range, or Contains, via the styles of the connecting lines.
+          graph nodes, and Superclass, Domain Of, Range, Type, or Contains, via the styles of the connecting lines.
           These style names are used in the explanation text that follows, below.
         </p>
         <p>
-          The diagram is roughly divided into left and right sections 
-          (although there are some common nodes; see later).
-          To make this description easier to understand, these will be referred to as the 
-          "Proof Section" and the "Verification Section". 
+          The diagram is roughly divided into (lower) left and (lower) right sections, and a top section.
+          These sections are connected by connecting lines of different types, and there is also one 
+          box, labeled as "multibase" and of a shape Datatype, that is shared by the two lower sections.
+          To make this description easier to understand, these sections will be referred to as the 
+          "Proof Section", "Verification Section", and "Verification Relationship Section", respectively. 
+        </p>
+        <p>
           Each of these sections has an ellipse at the top, styled as Class, 
-          and respectively labeled as "Proof" and "VerificationMethod".
+          and respectively labeled as "Proof", "VerificationMethod", and "VerificationRelationship".
```

Comment:
It seems odd to have a "Verification Section" with a "VerificationMethod" ellipse, and a "Verification Relationship Section" with a "VerificationRelationship" ellipse. I think it would be clearer to change (all references to) the "Verification Section" to the "Verification Method Section". (Or, as realized from later changes, revert it to the "VerificationMethod Section".)

Reply:
Or "Verification Method Section"? Ie, not using a camel-case term in the section heading...

Reply:
Oops, this was indeed one of your proposals...